posthog/temporal/data_imports/pipelines/pipeline/utils.py

```diff
@@ -398,6 +400,20 @@ def _update_last_synced_at_sync(schema: ExternalDataSchema, job: ExternalDataJob
     schema.save()
 
 
+# TODO: Implement this, needs to send a transactional email to the members of the team
+def _notify_revenue_analytics_that_sync_has_completed(schema: ExternalDataSchema) -> None:
+    if (
+        schema.source.source_type == ExternalDataSource.Type.STRIPE
+        and schema.source.revenue_analytics_enabled
+        and schema.source.created_at < datetime.now() - datetime.timedelta(minutes=5)
```

Comment:
Guessing this is checking for new sources? If so, this will often be False depending on when a user connects the source (e.g. time of day) and how big their stripe table is - paging Stripe takes _a long time_

Reply:
I wanna avoid sending emails if the sync is like really fast, because they'll probably be using the app still, and I don't wanna send them an email if it took like 2min. This is checking whether it took more than 5 minutes to run the sync